### ISSUE TITLE
lineplot markers=True silently ignored

### ISSUE THREAD

**Issue by aamster** [mwaskom/seaborn#2837]
When `markers=True` is passed to `lineplot`, it is silently ignored. 

The documentation should be updated to explain that `markers=True` is only relevant when `style` is passed

Ideally, `markers=True` would "just work" the way one would expect. If that's not possible, please issue a warning that it is only applied if `style` is passed

**Comment by mwaskom** [comment]
To clarify, the parameter is currently documented as applying to the `style` mapping:

> Object determining how to draw the markers for different levels of the `style` variable. Setting to True will use default markers, or you can pass a list of markers or a dictionary mapping levels of the style variable to markers. Setting to False will draw marker-less lines. Markers are specified as in matplotlib.

Are you saying that you would have benefitted from additional text here?

**Comment by aamster** [comment]
Yes, I would benefit from an improvement to the docs for this argument or a warning. What threw me off was 
> Setting to True will use default markers

so, I would expect that setting it to True would use default markers, but it only applies when  that `style` also be passed (I think?) . What I was expecting was what I got when I passed `marker='o'`

**Comment by khaldon** [comment]
Is that typo here `marks=True` or `marks='o'` doesn't work but works with `mark`?
